io.github.hermitdemschoenenleben.linien.yml

```diff
@@ -0,0 +1,112 @@
+app-id: io.github.hermitdemschoenenleben.linien
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: launcher.sh
+finish-args:
+  - --socket=x11
```

Comment:
You want `--device=dri` for GPU acceleration. Not crucial but will improve GUI performance.

Reply:
ah, great! Thanks, I just added this option